src/utils/stringUtils.js

```diff
@@ -1,3 +1,4 @@
+const lf = new Intl.ListFormat('en');
```

Comment:
This is only used once so it can be declared in the formatList filters function (between L37 and L38)